src/main/java/org/concordion/api/Resources.java

```diff
@@ -0,0 +1,46 @@
+package org.concordion.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Inlcudes resource files such as css, javascript, images, etc along with the generated specification.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Inherited
+public @interface Resources {
+
+	public enum InsertType {
+	   LINKED, EMBEDDED
+	}
+	
+	/**
+	 * Specify one or more files to add to the generated specification:
+	 * <ul>
+	 * 	<li>File names starting with '/' are relative to the root folder</li>
+	 * 	<li>File names not starting with '/' are relative to the class the annotation is applied to</li>
+	 * 	<li>Supports wild card characters '*' and '?' in the file name ("/images/*.png", "/css/*.*")</li>
+	 * 	<li>CSS and JS files are automatically added to the &lt;head&gt; section (see insertType parameter)
+	 * 	<li>All other file types must be manually added to the specification</li>
+	 * </ul>
+	 *  
+	 * @return Array of files
+	 */
+	public String[] value() default {};
```

Comment:
It was more so that I don't need to check for null values, although when I originally created it I did have a number of other options that made it more likely that you might not want to provide a value so the interface has been simplified a lot.